### ISSUE TITLE
When option to skip guilded comments/submissions is selected, it doesnt skip them.

### ISSUE THREAD

**Issue by doobersn** [j0be/PowerDeleteSuite#27]
Which kinda sucks. Please fix that, thanks!

**Comment by G0dis** [comment]
This is only half true. I do not see my gilded comments/posts under Comments or Posts. However, I still see them if I click on Awards Received. (This is "new" Reddit)
I chose to keep them, and they are still there. So it works, perhaps just not as expected because of Reddit.

**Comment by gu3st** [comment]
Yeah it'd be nice if it scraped https://www.reddit.com/user/<username>/gilded/ as well to find gilded posts.
